src/install_scripts/inno_setup_installer.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+if [ ! -f ./is.exe ]; then
+  wget -q http://www.jrsoftware.org/download.php/is.exe
+fi
+./is.exe //VERYSILENT //SUPRESSMSGBOXES //ALLUSERS
```

Comment:
How are we suppose to use this script?
I tried calling it from msys and the download works fine but then my screen blinks and the command exits without installing anything?
Is there something particular to do (if yes this should be documented)?

Reply:
By default it should open the Windows UAC warning dialog, if you accept, it should proceed to the installation. However, this UAC dialog may not show up and/or block the installation depending on your system settings. I added this script to check it was possible to install Inno Setup 6 from a script that we can run on AppVeyor. On AppVeyor, the UAC setting is defined so that it should not pop-up this UAC dialog and install Inno Setup silently, which is what we want.

Reply:
Thank you for the explanation, I confirm it is working fine on appveyor: https://ci.appveyor.com/project/DavidMansolino/webots/builds/25237289#L95